tests/dataflow/test_tiled_systolic.py

```diff
@@ -1,13 +1,23 @@
 # Copyright Allo authors. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+import os
+import pytest
 import re
 import allo
 from allo.ir.types import int32, Stream
 import allo.dataflow as df
 import allo.backend.hls as hls
 import numpy as np
 
+
+@pytest.fixture(scope="module", autouse=True)
+def setup_env():
+    os.environ["OMP_NUM_THREADS"] = "64"
+    yield
+    del os.environ["OMP_NUM_THREADS"]
```

Comment:
Are there any other ways to specify this? like a global fixture for pytest, instead of specifying it everywhere